package.json

```diff
@@ -9,8 +9,11 @@
     "url": "https://lightningpolar.com"
   },
   "main": "public/main/electron/index.js",
+  "engines": {
+    "node": ">=20.10.0"
+  },
```

Comment:
I think this should be set to the minimum version required for Polar to run. In my testing, v20.0.0 works but anything before that doesn't. I'd suggest changing this to `>=20.10.0`.

Reply:
You Mean >=20 ?

Reply:
lol.. yes. That is what I meant to write.